### PR TITLE
[Homework  #4] - CSS by Carlos Marchal

### PR DESCRIPTION
**Pull request by carlon702**
Hi Alexander, i did both class 4 and 5 in this pull request, please let me know if you need me to create a new branch for class 5.
Thanks

### CODE REVIEW

File: homeworks/carlos-marchal/04-css/styles/darkmode.css [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+    text-align: center;
+    margin: 25px auto 65px auto;
+}
+
+
+#darkModeButton {
+    border-radius: 100%;
+    background-color: #171F26;
+    transition: all linear 1000ms;
+    margin-top: 25px;
```
Comment (ababenko-solvd) [inline homeworks/carlos-marchal/04-css/styles/darkmode.css:36]: Remove redundant top margin

File: homeworks/carlos-marchal/04-css/styles/darkmode.css [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+
+.socialsLogo{
+    color: #A3ABB2;
+}
+
+
+
+.experienceStats {
+    width: 40%;
+    padding: 20px 2px 20px 2px;
```
Comment (ababenko-solvd) [inline homeworks/carlos-marchal/04-css/styles/darkmode.css:136]: Top and bottom are margins to distance between elements, but not for inner paddings

File: homeworks/carlos-marchal/04-css/index.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+        <button title="dark mode button" type="button" id="darkModeButton">
+            <i id="darkModeIcon" class="fa-regular fa-moon"></i>
+        </button>
+        
+    </header>
+
+    <main class="main">
+
+        <div class="portfolioPresentation">
```
Comment (ababenko-solvd) [inline homeworks/carlos-marchal/04-css/index.html:33]: Nice work. But you shoul use BEM approach for class naming in this HW. 
Rename all classes according to BEM pls

**Review by ababenko-solvd — CHANGES_REQUESTED** [review]

File: homeworks/carlos-marchal/04-css/index.html (hunk 3 above)
Comment (carlon702) [reply homeworks/carlos-marchal/04-css/index.html]: sorry alexander, i changed to BEM approach now, hope its okay

File: homeworks/carlos-marchal/04,05-css,advanced-css/index.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+                    <i class="fa-brands fa-twitter fa-xl portfolio-presentation__logo"></i>
+                    <i class="fa-brands fa-youtube fa-xl portfolio-presentation__logo"></i>
+
+                </div>
+            </div>
+        </section>
+
+
+        <div class="experience-stats">
+            <div class="experience-stats__experience-stats-container"> <p class="experience-stats__number">7</p> <p class="experience-stats__text"> Years of work <br> experience</p></div>
```
Comment (ababenko-solvd) [inline homeworks/carlos-marchal/04,05-css,advanced-css/index.html:51]: No need to repete experience-stats twice - experience-stats__container

**Review by ababenko-solvd — APPROVED** [review]

**Review by ababenko-solvd — APPROVED** [review]

**Review by ababenko-solvd — APPROVED** [review]

**Pull request merged**